astropy/utils/tests/test_decorators.py

```diff
@@ -23,9 +23,10 @@ class NewDeprecationWarning(AstropyDeprecationWarning):
 
 def test_wraps():
     """
-    Tests the compatibility replacement for functools.wraps which supports
+    Make sure functools.wraps supports
```

Comment:
Shouldn't we just delete the test? Not much point testing the python library...

Reply:
I don't remember why we had this. I just thought this would at least guard against regression if Python ever changed its behavior in the future. But if no one thinks it is useful, I can remove. Will wait for a second opinion first though.

Reply:
Astropy's `wraps` was removed in #7554 so :+1: with removing the test.

Reply:
OK, done!